src/app/icons/icons-sets/icons-tags.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export const ICONS_TAGS = {
+    "floppy": ["disk", "save"],
+    "image": ["graphic"],
+    "envelope": ["letter"],
+    "times": ["multiply"],
+    "power": ["on"],
+    "heart": ["love"],
+    "thumbs-up": ["like"],
+    "thumbs-down": ["dislike"],
+    "happy-face": ["emoji", "smile"],
+    "neutral-face": ["emoji", "smile"],
+    "sad-face": ["emoji", "smile"],
```

Comment:
why is `sad-face` and `neutral-face` tagged as `smile`?

Reply:
@adityarb88 aren't they called smile icons? 😛